Source/ablastr/utils/text/StreamUtils.cpp

```diff
@@ -0,0 +1,16 @@
+/* Copyright 2023
+ *
+ * This file is part of WarpX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+
+#include "StreamUtils.H"
+
+#include <limits>
```

Comment:
For `std::streamsize`:
```suggestion
#include <ios>
#include <limits>
```

Reply:
ah, yes, you are right